systemtests/tests/python-fd-ovirt-plugin-test/testrunner

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+#
+# This systemtest tests the oVirt plugin functionality
+# of the Bareos FD by using the supplied module
+#   BareosFdPluginOvirt.py
+#
+TestName="$(basename "$(pwd)")"
+export TestName
+
+JobName=backup-bareos-fd
+. ./environment
+. ${scripts}/functions
+
+${scripts}/cleanup
+${scripts}/setup
+
+curl -k -o ${BAREOS_CONFIG_DIR}/ovirt-ca.cert "https://$OVIRT_SERVER/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA"
+
+# Directory to backup.
+# This directory will be created by setup_data().
+# BackupDirectory="${tmp}/data"
+
+# Use a tgz to setup data to be backed up.
+# Data will be placed at "${tmp}/data/".
+# setup_data
+
+start_test
+
+cat <<END_OF_DATA >$tmp/bconcmds
+@$out /dev/null
+messages
+@$out $tmp/log1.out
+setdebug level=100 storage=File
+label volume=TestVolume001 storage=File pool=Full
+run job=$JobName level=Full yes
+status director
+status client
+status storage=File
+wait
+messages
+@#
+@# now do a restore
+@#
+@$out $tmp/log2.out
+wait
+restore client=bareos-fd fileset=OvirtTest where=$tmp/bareos-restores pluginoptions=python:local=yes select all done
```

Comment:
Yes, currently the systemtest only tests restore to local disk image file.
Restore into VM could be added easily, but verifying that such a restore did really work correctly will be more complicated, as it would probably require accessing the restored VM via ssh.